pkg/asset/installconfig/baremetal/baremetal.go

```diff
@@ -0,0 +1,256 @@
+// Package baremetal collects bare metal specific configuration.
+package baremetal
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/openshift/installer/pkg/ipnet"
+	"github.com/openshift/installer/pkg/types/baremetal"
+	baremetaldefaults "github.com/openshift/installer/pkg/types/baremetal/defaults"
+	"github.com/openshift/installer/pkg/validate"
+	"gopkg.in/AlecAivazis/survey.v1"
+	"gopkg.in/AlecAivazis/survey.v1/terminal"
+)
+
+// Platform collects bare metal specific configuration.
+func Platform() (*baremetal.Platform, error) {
+	var libvirtURI, provisioningNetworkCIDR, clusterProvisioningIP, bootstrapProvisioningIP, externalBridge,
+		provisioningBridge, provisioningNetworkInterface, apiVIP, ingressVIP, bootstrapOSImage, clusterOSImage string
+	var hosts []*baremetal.Host
+
+	if err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Libvirt URI",
+				Help:    "The identifier for the libvirtd connection.",
+				Default: baremetaldefaults.LibvirtURI,
+			},
+			Validate: uriValidator,
+		},
+	}, &libvirtURI); err != nil {
+		return nil, err
+	}
+
+	if err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Provisioning Network CIDR",
+				Help:    "The network used for provisioning.",
+				Default: "172.22.0.0/24",
```

Comment:
Not sure, lean towards leave this one -- but drop everything else calculated off it.